spec/forms/select_email_form_spec.rb

```diff
@@ -29,14 +29,28 @@
     end
 
     context 'with an invalid email id' do
-      let(:selected_email_id) { '' }
+      context 'with a blank email id' do
+        let(:selected_email_id) { nil }
```

Comment:
Can you change this back to `''` as modified in #11549 ? I'd made the change intentionally to more accurately represent the value we'd expect to receive on the backend if the user didn't make a selection on the frontend (e.g. if JavaScript is disabled).

```suggestion
        let(:selected_email_id) { '' }
```

Reply:
Oops. Glad you noticed that! Reverted that value.